desktop/apps/consignments2/client/actions.js

```diff
@@ -46,6 +54,7 @@ export function addImageToUploadedImages (fileName, src) {
 export function chooseArtistAndAdvance (value) {
   return (dispatch) => {
     dispatch(updateArtistId(value._id))
+    dispatch(updateArtistName(value.name))
```

Comment:
Just curious -- why not a single `updateArtist` action that updates a nested `artist` object?